src/resources/MachineLearning/ModelInformation/ModelInformationInterfaces.ts

```diff
@@ -209,9 +209,39 @@ export interface BuildingStatsSS {
     headerCount: number;
     meanSnippetLength: number;
 }
+
+export interface SnippetsPerDocument {
+    min: number;
+    max: number;
+    mean: number;
+    median: number;
+    p95: number;
+}
+
+export interface StatsSS {
```

Comment:
"sSS" is a little much i think we can use the whole "SmartSnippets" when the name isn't super long.

Reply:
🐍 ssSStopTheSsssSS 🐍